app/editor/src/features/content/form/components/tags/Tags.tsx

```diff
@@ -64,14 +52,9 @@ export const Tags: React.FC<ITagsProps> = ({ selectedExternal }) => {
       .map((tag) => tag);
   };
 
-  const addManualTags = (selectedTags: any) => {
-    const extCodes = externalTags.map((x: ITagModel) => x.code);
-    const manualTags = convertTags(selectedTags).filter(
-      (x: ITagModel) => !extCodes.includes(x.code),
-    );
-    setManualTags(manualTags);
-    const newTags = manualTags.concat(externalTags);
-    setFieldValue('tags', newTags);
+  const addTags = (selectedTags: any) => {
```

Comment:
This rings a bell for me.  Was updating the tags via the document meant to update the tag selector **AND ALSO** vice-a-versa?

Reply:
Parsed tags are added to the list, and manually selected tags are added to the list.  If they are removed in the story Summary/Body they don't get removed from the dropdown regrettably.  This may be an issue.

Reply:
You can now remove tags through removing the parsed values.